### PR TITLE
added userhistory

### PR DESCRIPTION
**Pull request by AchrafBoufer**
Added a new method to the controller to handle operations. Fixed the UI but didn't make it infinite yet, also didn't hook up the Matrix side of our program yet.

### CODE REVIEW

**Review by mohtashim-ali — CHANGES_REQUESTED** [review]
Calling Entity in Controller may violate Clean Architecture

File: src/main/java/Controller/CalculatorController.java [hunk 1]
```diff
@@ -11,29 +14,42 @@
...
-            matrix.computeMatrix();
+        return false;
+    }
+
+    public void handleOperations(String input) throws ExpressionException, IOException {
+        if(this.choice == 1) {
+            BuildExpression buildExpression = new BuildExpression();
+            buildExpression.setInput(input);
+            StringBuilder fixed_input = buildExpression.makeExpression();
+            userHistory.addToHistory(fixed_input.toString(), "2:00"); // Temporary time
```
Comment (krhobeck) [inline src/main/java/Controller/CalculatorController.java:46]: Comment out temporary line

File: src/main/java/UseCase/Accounts/CurrentUser.java [hunk 2]
```diff
@@ -4,10 +4,10 @@
 
 public class CurrentUser {
```
Comment (krhobeck) [inline src/main/java/UseCase/Accounts/CurrentUser.java:5]: should also have a getCurrentUser function

**Review by krhobeck — CHANGES_REQUESTED** [review]
Look into these extra changes aswell

**Pull request merged**